docs/contributors/readme.md

```diff
@@ -2,7 +2,7 @@
 
 Welcome to the Gutenberg Project Contributors Guide.
 
-The following guidelines are in place to create consistency across the project and the numerous contributors. See the [Contributing Documentation](https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md) for technical details around setup, and submitting your contributions.
+The following guidelines are in place to create consistency across the project and the numerous contributors. See the [Contributing Documentation](https://github.com/WordPress/gutenberg/blob/master/docs/contributors/develop.md) for technical details around setup, and submitting your contributions.
```

Comment:
The existing link is correct and should not be updated as well.

Reply:
I am reversing this change now.